_posts/2018-08-08-release-1.6.0.md

```diff
@@ -0,0 +1,160 @@
+---
+layout: post
+title:  "Apache Flink 1.6.0 Release Announcement"
+date:   2018-08-08 18:30:00
```

Comment:
AFAIK we wanted to keep use a consistent time (12:00:00), but I don't really know why. Just saw this comment in another PR.

Reply:
Alright, will update the blog post accordingly.